src/main/java/emissary/util/PlaceComparisonHelper.java

```diff
@@ -0,0 +1,185 @@
+package emissary.util;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.List;
+
+import emissary.config.Configurator;
+import emissary.core.IBaseDataObject;
+import emissary.core.IBaseDataObjectHelper;
+import emissary.place.ServiceProviderPlace;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.Validate;
+
+/**
+ * Helper class to use during development of a major refactoring or replacement of a Place.
+ */
+public class PlaceComparisonHelper {
+
+    private PlaceComparisonHelper() {}
+
+    private static final String FMT_COMPARISON_IDENTIFIER = "Comparison[%s==%s]";
+    // Config key to use in Place configuration to define another place to compare
+    public static final String CFG_PLACE_TO_COMPARE = "PLACE_TO_COMPARE";
+
+    /**
+     * Given a config which contains a 'PLACE_TO_COMPARE' directive, instantiate the referenced place with its usual
+     * configuration
+     * 
+     * @param configG from an existing place which is scanned for PLACE_TO_COMPARE, which should reference a place that the
+     *        developer wants to compare against
+     * @return the instantiated place if configured
+     * @throws ClassNotFoundException if we can't find the other place
+     * @throws SecurityException if we can't access the String constructor for the other place
+     * @throws NoSuchMethodException if there isn't an accessible constructor for the other place
+     * @throws InvocationTargetException if the underlying constructor throws an exception
+     * @throws IllegalArgumentException if the number of actual and formal parameters differ; if an unwrapping conversion
+     *         for primitive arguments fails; or if, after possible unwrapping, a parameter value cannot be converted to the
+     *         corresponding formal parameter type by a method invocation conversion; if this constructor pertains to an
+     *         enum class
+     * @throws IllegalAccessException if the Constructor object is enforcing Java language access control and the underlying
+     *         constructor is inaccessible.
+     * @throws InstantiationException if the class that declares the underlying constructor represents an abstract class.
+     */
+    public static ServiceProviderPlace getPlaceToCompare(final Configurator configG) throws ClassNotFoundException,
```

Comment:
If the caller doesn't care about the different cases they can catch and handle Exception, but I'll remove IllegalArgumentException since it's a RuntimeException.